tests/flytekit/unit/core/test_type_engine.py

```diff
@@ -749,18 +749,19 @@ class TestFileStruct(object):
 
 def test_structured_dataset_in_dataclass():
     df = pd.DataFrame({"Name": ["Tom", "Joseph"], "Age": [20, 22]})
+    People = Annotated[StructuredDataset, "parquet", kwtypes(Name=str, Age=int)]
```

Comment:
nit: preserve the bare `StructuredDataset` case.

Do we need to test other `Annotated` combinations? like `Annotated[StructuredDataset, kwtypes(...)]` without specifying the file format?

Reply:
yeah, updated it